ingestion_tools/scripts/transform_ingestion_configs.py

```diff
@@ -35,6 +36,65 @@ def has_no_sources(data: list[dict[str, Any]] | dict[str, Any]) -> bool:
     return isinstance(data, dict) or not any(row.get("sources") for row in data)
 
 
+def rawtilts_to_alignments(data: dict) -> None:
```

Comment:
Nit for this function in general -- in Python, I think the (return early)[https://openfolder.sh/returning-early] pattern is best practice. There are several points in this function where we do a check for a value and then nest more logic inside that only gets executed in that case, and yet there's no `else` condition - we could have just returned from this function (or added a `continue/break` to our loop) if the condition wasn't met. The code here would be shorter and less deeply nested (thus: easier to read) if it were refactored in this way